### PR TITLE
AC Convert Test - LPD-30751 | master

### PR DESCRIPTION
**Pull request by ViniLopes20**
https://liferay.atlassian.net/browse/LPD-30751

### CODE REVIEW

**Comment by ViniLopes20** [comment]
ci:test:analytics-cloud

**Comment by ViniLopes20** [comment]
ci:test:sf

File: modules/test/playwright/tests/osb-faro-web/segments.spec.ts [hunk 1]
```diff
@@ -622,3 +623,226 @@ test('Segment Composition shows Active and Known individuals', async ({
...
+			await waitForLoading(page);
+		});
+
+		await test.step('Check the segment member count in the membership', async () => {
+			await navigateTo({
+				page,
+				pageName: 'Membership',
+			});
+
+			const knownMembersCount = page
```
Comment (marcosapmf) [inline modules/test/playwright/tests/osb-faro-web/segments.spec.ts:842]: inline

**Comment by ViniLopes20** [comment]
ci:forward

File: modules/test/playwright/tests/osb-faro-web/segments.spec.ts (hunk 1 above)
Comment (ViniLopes20) [reply modules/test/playwright/tests/osb-faro-web/segments.spec.ts]: Done

**Pull request closed without merging**